### DISCUSSION TITLE
SmartNotes CE bug posting and suggestions topic.

### DISCUSSION THREAD

**Discussion by randomguy70** [randomguy70/SmartNotesCE#3] [Q&A] [answered]
If you find any bugs, or have any ideas for improvement, then please post below! Thank you!

**Comment by randomguy70** [comment]
btw, don't try this program on your physical calculator yet! it hardly even runs!

**Comment by mateoconlechuga** [comment]
Can you make this an actual repository, not a template

**Comment by randomguy70** [comment, marked as answer]
Ok.
